### ISSUE TITLE
Updating repository when new perl modules are needed

### ISSUE THREAD

**Issue by samseaver** [ModelSEED/ModelSEED#34]
I successfully installed the ModelSEED repository using Paul's install script, but when I ran a git pull just now, I find that I need yet another Perl module in my system.  This is fine for me to handle, but for other users, there should be an update script that works like Paul's install script in that it also updates the necessary Perl modules.  There could even be an 'update' flag in the same Install script.

I supposed this may not be an issue if the set of Perl modules required were to be settled on before public release, but if we do expect some changes in the modules needed, then there should be an accompanying script that makes this easier for other users.

**Comment by pfrybar** [comment]
Yep, I'm working on an 'update' command that will update the ModelSEED and MFAToolkit, including installing new Perl dependencies. It will work for users who install using the install script.

**Comment by samseaver** [comment]
Just saw your older issue #4

**Comment by mdejongh** [comment]
Please send instructions on how to run Paul's install script (or tell me
where the documentation is). I'm meeting with my students this morning and
want to get them launched into this brave new world. Thanks,

Matt

On Wednesday, August 29, 2012, samseaver wrote:

> I successfully installed the ModelSEED repository using Paul's install
> script, but when I ran a git pull just now, I find that I need yet another
> Perl module in my system. This is fine for me to handle, but for other
> users, there should be an update script that works like Paul's install
> script in that it also updates the necessary Perl modules. There could even
> be an 'update' flag in the same Install script.
> 
> I supposed this may not be an issue if the set of Perl modules required
> were to be settled on before public release, but if we do expect some
> changes in the modules needed, then there should be an accompanying script
> that makes this easier for other users.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/ModelSEED/ModelSEED/issues/34.

## 

Matt DeJongh, Ph.D.
Computer Science Department
Hope College, Holland, MI 49423
(616) 395-7429

**Comment by pfrybar** [comment]
Hey Matt, I'm working on documentation for the wiki, but in the meantime I'll send you an email with the instructions.
